action.yml

```diff
@@ -44,8 +44,13 @@ inputs:
     description: 'Azure connected cluster name'
     required: false
     default: ''
+  environment:

```

Comment:
Missed this earlier. Is this needed for connected-cluster? This was required in the login action for some specific reasons. By why do we need this here?

Reply:
This is needed for connected cluster as well. az login would need to be done in the cloud where cluster is present.

Reply:
then we are simply introducing a dependency for this action. And plus we still need code from az login action to run other az commands like `az extension add`

Reply:
Once you're logged in, you can run any az command. And the code to run az commands should be very minimal. We had this conversation when I was adding the connected cluster code to this action. azlogin is meant to be used as a dependency on other az actions.